### DISCUSSION TITLE
Mac OS 10.13.6 running Artisan 1.4.0 com port serial port error

### DISCUSSION THREAD

**Discussion by edwinloh68** [artisan-roaster-scope/artisan#647] [Q&A]
I am using a macbook air 2017 running mac OS version 10.13.6. I downloaded the latest version of artisan but it would not install due to my older OS version. I then downloaded artisan 1.4.0. I manage to install and configure artisan but when I try to connect my home made roaster which is using Fuji PID via USB to my mac, it showed com port error. I set the com port to 4. How do I setup the com port serial port correctly on the mac.

I manage to get it working on my PC which I can see and set the port using system config on Windows. Thanks for you kind help.

**Comment by poundy** [comment]
Friends don't let friends use old, unsupported software. Artisan 1.x is long dead. If you want help, you'd need to get to a recent MacOS that can run current release of Artisan. Sorry, that old version of Artisan is long replaced for a reason. Don't bother with it, just fix the OS issue

**Reply by MAKOMO** [reply]
You need to choose the correct serial port in the Ports Configuration dialog tab ET/BT (menu Config >> Ports, first tab). Disconnect the device and check the entries in the Comm Port popup.  Connect the device again and check again the entries in the Comm Port popup. The new entry should be the one of the Fuji PID. On macOS, USB ports are listed as /dev/tty.... You can as well check in your System Report for the assigned serial port while the PID is connected and just type that /dev/tty... port into the Comm Port field. However, I agree. Update to a current system and ensure that you use the latest Artisan version.
